lib/usecases/patchApplications.js

```diff
@@ -1,16 +1,33 @@
 const ObjectsToCsv = require('objects-to-csv');
+const DB_TABLE_GRANT_PREFIXES = [
+  'lrsg_closed_businesses',
+  'lrsg_open',
+  'lrsg_sector',
+];
 
-const patchApplications = ({ getDbInstance }) => async ({ author }) => {
+const patchApplications = ({ getDbInstance }) => async ({
+  author,
+  grant_type,
+}) => {
   const db = await getDbInstance();
+
+  if (DB_TABLE_GRANT_PREFIXES.includes(grant_type) === false) {
+    throw new Error('Invalid grant type');
+  }
+
   const query = `
     WITH exported AS
     (
         UPDATE
             application_assessment
         SET
-            application_state_id = 16 -- Exported for Payment
+            ${grant_type}_payment_exported = TRUE -- Mark as Exported for Payment
```

Comment:
Do we want to set the application status to `exported for payment` as well?

Reply:
No, I discussed this with Miranda and she just wanted status left alone.